controllers/articleRatingController.js

```diff
@@ -0,0 +1,61 @@
+const rating = require('../lib/modelManagers/articleRatingModel'),
+  response = require('../lib/utils/responses'),
+  {
+    OK_CODE,
+    RESOURCE_CREATED_CODE,
+    SERVER_ERROR_MESSAGE,
+    SERVER_ERROR_CODE,
+    NOT_FOUND_CODE,
+    GET_USER_RATING,
+    GET_USER_RATING_ERROR,
+    RATING_CREATED
+  } = require('../constants/index');
+
+/**
+ * @description Article Rating Controller Class.
+ */
+class articleRating {
+  /**
+  * Creates or Update Rating.
+  * @param {object} req Request Object.
+  * @param {object} res Response Object.
+  * @returns {object} Response with status, success true or false, and message.
+  */
+  static async createRating(req, res) {
+    const newRating = {
+      reviewerId: req.body.reviewerId,
+      articleId: req.body.articleId,
+      rating: req.body.rating
+    };
+
+    try {
+      const result = await rating.createRating(newRating);
+      await rating.addRatingToArticle(req.body.articleId);
```

Comment:
This function should be in the article model manager

Reply:
Noted